recipes/motus/meta.yaml

```diff
@@ -0,0 +1,25 @@
+package:
+  name: motus
+  version: "1.0"
+
+source:
+  fn: mOTUs.Linux64bits.tar.gz
+  url: http://www.bork.embl.de/software/mOTU/share/mOTUs.Linux64bits.tar.gz
```

Comment:
Oh, even worse, the binaries are includes in the perl file as binary blob - as text - this is ... omg.